src/main/drivers/bus_spi.h

```diff
@@ -31,6 +31,7 @@
 #define SPI_IO_AF_MOSI_CFG    IO_CONFIG(GPIO_Mode_AF_PP,       GPIO_Speed_50MHz)
 #define SPI_IO_AF_MISO_CFG    IO_CONFIG(GPIO_Mode_IN_FLOATING, GPIO_Speed_50MHz)
 #define SPI_IO_CS_CFG         IO_CONFIG(GPIO_Mode_Out_PP,      GPIO_Speed_50MHz)
+#elif defined(NOPROCESSOR)
```

Comment:
Rather than introduce a new `#define` I think it would be better to just remove the error condition, eg
```
...
#define SPI_IO_CS_CFG         IO_CONFIG(GPIO_Mode_Out_PP,      GPIO_Speed_50MHz)
#endif
```

After all, it is not the job of `bus_spi.h` to police the build.